### PR TITLE
Added NTAG424 compatibility

### PR DESCRIPTION
**Pull request by RiverRuby**
Sent tapIds for Devcon to Andrew, there are 2 test ones in seed.ts

### CODE REVIEW

File: apps/backend/src/lib/controller/chip/managed/util.ts [hunk 1]
```diff
@@ -48,18 +50,54 @@ export async function getChipFromTapParams(
...
-    });
+    while (retries < maxRetries) {
+      try {
+        const response = await fetch(
+          `http://ec2-13-215-189-29.ap-southeast-1.compute.amazonaws.com:9091/api/validate?e=${encryptedChipId}`,
+          {
+            method: "GET",
+            headers: {
+              Authorization:
+                "Bearer 466b9db251785c2cc815a327ca4547e06222cc24af377cbb0729d543896ea599",
```
Comment (AndrewCLu) [inline apps/backend/src/lib/controller/chip/managed/util.ts:67]: env var? lol

**Review by AndrewCLu — APPROVED** [review]

**Comment by AndrewCLu** [comment]
tested, working

**Review by AndrewCLu — APPROVED** [review]

File: apps/backend/src/lib/controller/chip/managed/register/index.ts [hunk 2]
```diff
@@ -17,17 +17,24 @@ ManagedChipClient.prototype.RegisterChip = async function (
...
-  // Generate a new public/private key pair
-  const { signingKey, verifyingKey } = generateSignatureKeyPair();
+  let updateSigningKey: string | undefined;
+  let updateVerifyingKey: string | undefined;
+
+  // Generate a new public/private key pair if none exists
+  if (!chip.chipPrivateKey || !chip.chipPublicKey) {
+    const { signingKey, verifyingKey } = generateSignatureKeyPair();
+    updateSigningKey = signingKey;
+    updateVerifyingKey = verifyingKey;
```
Comment (RiverRuby) [inline apps/backend/src/lib/controller/chip/managed/register/index.ts:27]: Need to add new Cursive EdDSA signature

**Review by RiverRuby — COMMENTED** [review]
Need to test this EdDSA signature works for proof generation, so will also have e2e coSNARK generation as part of this PR

**Pull request merged**